core/deployment/src/main/java/io/quarkus/deployment/IsDockerWorking.java

```diff
@@ -21,25 +23,26 @@
 
 import io.quarkus.deployment.console.StartupLogCompressor;
 import io.quarkus.deployment.util.ExecUtil;
+import io.quarkus.runtime.LaunchMode;
 
 public class IsDockerWorking implements BooleanSupplier {
 
     private static final Logger LOGGER = Logger.getLogger(IsDockerWorking.class.getName());
+    public static final int DOCKER_HOST_CHECK_TIMEOUT = 1000;
+    public static final int DOCKER_CMD_CHECK_TIMEOUT = 3000;
 
     private final List<Strategy> strategies;
 
-    public IsDockerWorking() {
-        this(false);
-    }
-
-    public IsDockerWorking(boolean silent) {
+    public IsDockerWorking(LaunchMode launchMode) {
```

Comment:
IMHO the explicit 'silent' param is better, there may be some dev and test use cases where we actually want to see the docker output.

Reply:
👍 I changed it back to `slient`.